templates/shared/issuelist.tmpl

```diff
@@ -118,6 +118,32 @@
 							</span>
 						</span>
 					{{end}}
+					{{if and $.BlockingDependenciesMap (index $.BlockingDependenciesMap .ID)}}
+						<div class="flex-text-inline gt-gap-0">
+							<strong class="gt-mr-2">{{ctx.Locale.Tr "repo.issues.dependency.blocks_short"}}:</strong>
```

Comment:
Please don't construct a string like that, simply declare a new string that defaults to `Depends on: %s` in English.

Reply:
Ok, but I think the ` %s` is useless, because I need links?

Reply:
`Depends on: %s` is technically correct because of RTL languages but getting it to work will be ugly. One way I know:

```handlebars
{{ctx.Locale.Tr "repo.issues.dependency.blocks_short" (printf `<a href="%[1]s"><b>%[2]s</b></a>` ($link|Escape) ($content|Escape)) | Safe}}
```

But this will become very tricky with the logic that follows. Personally I wouldn't mind if we move this HTML construction to a helper function, at least the parts that generates the links could be universally usable.

Reply:
I hope its better now